packages/actor-tests/bedrock/deposit-eth.ts

```diff
@@ -1,71 +1,74 @@
-import { utils, Wallet, providers, constants } from 'ethers'
+import { utils, Wallet, constants } from 'ethers'
 import {
   CrossChainMessenger,
   ETHBridgeAdapter,
   StandardBridgeAdapter,
 } from '@eth-optimism/sdk'
-import { predeploys } from '@eth-optimism/contracts-bedrock'
+import { predeploys } from '@eth-optimism/contracts-bedrock/dist'
```

Comment:
Does this not work without `dist`?

Reply:
Nope, it didn't work without dist :(.

Reply:
Maybe @tynes knows how to fix it? It was trying to find an index file that didn't exist.

Reply:
Try updating to the latest release? https://github.com/ethereum-optimism/optimism/blob/6f66c71f4163dc28f142ef87285631f4b64d2f05/packages/contracts-bedrock/package.json#L3

Your `package.json` refers to an older release. Other packages don't need to import from dist like this when working with this package?

Reply:
Fixed!